src/_pytest/debugging.py

```diff
@@ -63,6 +76,24 @@ def fin():
     config._cleanup.append(fin)
 
 
+@hookimpl(hookwrapper=True)
+def pytest_pyfunc_call(pyfuncitem):
+    if immediately_break:
+        pytestPDB.set_trace(set_break=False)
+        testfunction = pyfuncitem.obj
+        pyfuncitem.obj = pdb.runcall
+        if pyfuncitem._isyieldedfunction():
+            pyfuncitem.args = [testfunction, pyfuncitem._args]
+        else:
+            if "func" in pyfuncitem._fixtureinfo.argnames:
```

Comment:
this is already availiable in request - the fixture should be named pytest_test_function and be availiable in any case